criticality_score/generate.py

```diff
@@ -135,8 +147,9 @@ def main():
                 output = run.get_repository_stats(repo)
                 break
             except Exception as exp:
-                print(
-                    f'Exception occurred when reading repo: {repo_url}\n{exp}')
+                msg = f'Exception occurred when reading repo: {repo_url}\n{exp}'
+                print(msg)
```

Comment:
This is not the right use of logging methods, to just log errors and not anything else. I think you can add logging handler, but then convert all the prints to use that logging pipelines (regular print should be info level). Then log file will have all log data, info and errors. and you can search for errors using error log level.

Reply:
I see, since logger also prints the messages to the console.

Updates in last commit:
- Move logger to global
- Replace print with logger.info
- Set loglevel to info

I also tried to share the logger with `run` script. The only way that I could do it was to pass it as an argument in each function. e.g. `def get_repository(url, logger):`

I assume that there's a better way to do it, so I'm skipping that one for now.